packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -44,39 +52,29 @@ const DECODE_CONTRACT_CALL_RESULT_ABI = [
   },
 ];
 
-// TODO: get these from terms
-const AAVE_POOL_ADDRESS = '0xccEa5C65f6d4F465B71501418b88FBe4e7071283';
-const USDC_TOKEN_ADDRESS = '0xCaC7Ffa82c0f43EBB0FC11FCd32123EcA46626cf'; // not circle USDC
-
-export const supportedEVMChains: CaipChainId[] = [
-  'eip155:43114', // Avalanche
-  'eip155:8453', // Base
-  'eip155:1', // Ethereum
-];
-
 const TypeShape = M.or(...keys(YieldProtocol));
-export const ChainShape = M.or(...values(PositionChain));
+export const ChainShape = M.string(); // TODO: remove it
 const PositionShape = M.splitRecord({}); // TODO
 
 const KeeperI = M.interface('keeper', {
-  add: M.call(
-    TypeShape,
-    M.or(ChainShape),
+  addAavePosition: M.call(M.or(ChainShape)).returns(),
+  addCompoundPosition: M.call(M.or(ChainShape)).returns(),
+  addUSDNPosition: M.call(
+    M.string(),
     M.remotable('OrchestrationAccount'),
-  ).returns(M.number()),
+  ).returns(),
   getPositions: M.call(TypeShape, ChainShape).returns(M.arrayOf(PositionShape)),
-  getAccount: M.call(M.number(), TypeShape).returns(
-    M.remotable('OrchestrationAccount'),
-  ),
-  getRemoteAccountAddress: M.call(M.number()).returns(
-    M.or(M.string(), M.undefined()),
-  ),
-  sendGmp: M.call(M.remotable('Seat'), M.record()).returns(M.promise()),
+  getAccount: M.call(TypeShape).returns(M.remotable('OrchestrationAccount')),
 });
 
 const HolderI = M.interface('Holder', {
-  supplyToAave: M.call(M.remotable('Seat')).returns(M.promise()),
-  withdrawFromAave: M.call(M.remotable('Seat')).returns(M.promise()),
+  supplyToAave: M.call(M.record()).returns(VowShape),
+  withdrawFromAave: M.call(M.record()).returns(VowShape),
+  setupGmpLCA: M.call(M.remotable('OrchestrationAccount')).returns(),
```

Comment:
making an LCA has to be done in a flow; might as well do it in `openPosition`. See #11505